src/components/Assesment/Assesment.jsx

```
@@ -489,36 +581,81 @@ export const ProfileHeader = ({
+                    width: "40px",
+                    height: "40px",
+                    borderRadius: "50%",
+                    backgroundColor: "#007bff",
+                    color: "white",
+                    fontSize: "24px",
+                    cursor: "pointer",
+                    boxShadow: "0 4px 6px rgba(0, 0, 0, 0.1)",
+                    transition: "background-color 0.3s, transform 0.3s",
                   }}
+                  onMouseEnter={(e) =>
+                    (e.target.style.transform = "scale(1.1)")
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)